lib/web_ui/lib/src/engine/platform_dispatcher/view_focus_binding.dart

```diff
@@ -67,36 +62,58 @@ final class ViewFocusBinding {
   });
 
   void _handleFocusChange(DomElement? focusedElement) {
+    final int? lastViewId = _lastViewId;
     final int? viewId = _viewId(focusedElement);
-    if (viewId == _lastViewId) {
+    if (viewId == lastViewId) {
       return;
     }
+
     final ui.ViewFocusEvent event;
-    if (viewId == null) {
-      event = ui.ViewFocusEvent(
-        viewId: _lastViewId!,
-        state: ui.ViewFocusState.unfocused,
-        direction: ui.ViewFocusDirection.undefined,
-      );
-    } else {
+    if (viewId != null) {
       event = ui.ViewFocusEvent(
         viewId: viewId,
         state: ui.ViewFocusState.focused,
         direction: _viewFocusDirection,
       );
+    } else {
+      event = ui.ViewFocusEvent(
+        viewId: lastViewId!,
```

Comment:
What if `lastViewId` is null? (That's why you flipped this if/else, right? ;) )

Reply:
Good catch. Reverted the if/else.